src/plugin.js

```diff
@@ -206,6 +206,9 @@ const contribAdsPlugin = function(options) {
 
   player.ads.adMacroReplacement = adMacroReplacement.bind(player);
 
+  // Expose snapshot tools for testing
+  player.ads._snapshot = snapshot;
```

Comment:
Why is this not already available to tests via player.ads.snapshot? Can we import it into the test file instead?